library/core/src/num/int_macros.rs

```diff
@@ -2000,5 +2000,176 @@ macro_rules! int_impl {
         pub const fn max_value() -> Self {
             Self::MAX
         }
+
+        /// Adds to the current value, returning the previous value.
+        ///
+        /// This operation wraps around on overflow.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// #![feature(int_fetch_ops)]
+        #[doc = concat!("let foo: ", stringify!($SelfT), " = 0;")]
+        /// assert_eq!(foo.fetch_add(10), 0);
+        /// assert_eq!(foo, 10);
+        /// ```
+        #[inline(always)]
```

Comment:
Does this need `always`? `#[inline]` seems fine.

Reply:
I'll change this.